src/server/controllers/match.controller.js

```diff
@@ -69,7 +74,7 @@ function checkExpertiseFilter(arr, val) {
 
 function arrayContainsElement(arr, val) {
   return arr.some((arrVal) => {
-    return val === arrVal;
+    return val == arrVal;
   });
```

Comment:
http://eslint.org/docs/rules/eqeqeq

I thought '===' is always safer than '==' in any cases. 
If you think it guarantees safe comparison even in further change, that will be okay.